emperor/support_files/js/sceneplotview3d.js

```diff
@@ -525,19 +525,7 @@ define([
     decomp = this.decViews[firstKey].decomp;
 
     this._dimensionsIterator(function(start, end, index) {
-      // when the labels get too long, it's a bit hard to look at
```

Comment:
Moved this into the model that way I could pluck it off in the view (I went back and forth on where the rendering code would live a few times, so this might just be a remnant of an old iteration).